### PR TITLE
fix clickable url output

### PR DESCRIPTION
**Pull request by tkjef**
links weren't formatted correctly

### CODE REVIEW

File: plugin/vim-github-links.vim [hunk 1]
```diff
@@ -1,10 +1,10 @@
 " My function to create github links for the current line of code
 function! GithubLink()
   let git_branch = system("git status | awk '/On branch/ {print $3}'| ruby -e 'print gets.strip'")
-  let repo_name = system("git config --get remote.origin.url | ruby -e \"print gets.gsub(/.*@/, '').gsub(':', '/').gsub('.git', '').strip\"")
+  let repo_name = system("git config --get remote.origin.url | ruby -e \"print gets.gsub(/.*@/, '').gsub('.git', '').strip\"")
```
Comment (baroldgene) [inline plugin/vim-github-links.vim]: When running this command (git config --get remote.origin.url) I get this output:
git@git.innova-partners.com:cmm/CMM_PHP
The colon still needs to be removed and the https added.  Unless I'm missing something.

**Comment by tkjef** [comment]
The current formatting adds an extra "http" and replaces the ":" in the "http://" with another "/" which makes the links error out when clicked on.

File: plugin/vim-github-links.vim (hunk 1 above)
Comment (tkjef) [reply plugin/vim-github-links.vim]: $ git config --get remote.origin.url
https://github.com/tkjef/vim-github-links.git

is what i get. What os are you on? I've tested on Ubuntu 16.04 and Mac High Sierra.

**Comment by baroldgene** [comment]
Can you tell me what the output of this command is on your system?
`git config --get remote.origin.url`

When I run this I don't have an http:// to replace.  Works as desired.

To be clear I'm not saying you don't have a bug here.  Just want to make sure it works on both systems not just one or the other.

**Comment by tkjef** [comment]
Looks like the issue is what the url was originally cloned from.

**Comment by tkjef** [comment]
I'll add a conditional to check for that.

**Pull request closed without merging**